Formula/haskell-language-server.rb

```diff
@@ -0,0 +1,36 @@
+class HaskellLanguageServer < Formula
+  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  homepage "https://github.com/haskell/haskell-language-server"
+  url "https://github.com/haskell/haskell-language-server.git",
+      tag:      "1.0.0",
+      revision: "4cd1cf934638881e52b3eba9f70157a4b799c0e9"
```

Comment:
```suggestion
  url "https://github.com/haskell/haskell-language-server/archive/1.0.0.tar.gz"
  sha256 "14e28d6621d029f027fae44bc4a4ef62c869dab24ff01b88a2e51e6679cbff6c"
```
Can we use the release tarball for this?

Reply:
Sure, didn't notice it was possible 😅